Lib/test/test_logging.py

```diff
@@ -50,6 +49,7 @@
 from test.support.logging_helper import TestHandler
 import textwrap
 import threading
+import asyncio
```

Comment:
Is it okay to leave this imported here now that I've added the test which temporarily removes it?

Reply:
Sure.